reference/auth/models/keyid.v1.yaml

```
@@ -0,0 +1,4 @@
+title: Key Id
+description: Base64 encoded key identifier received from apple. The Key Id is some shortened data, usually a hash, used to identify the longer actual key.
```

nit

```suggestion
description: An opaque identifier received from Apple.
```

As it's used here "opaque" is understood to indicate that whatever data we receive, we shouldn't try to decode it or understand it or anything. This is commonly done because its creator is reserving the right to change its structure or meaning at any time.